src/cmd/service.go

```diff
@@ -85,6 +85,11 @@ var listServiceCmd = &cobra.Command{
 		cobra.CheckErr(err)
 		if isJsonOutput() {
 			common.JsonPrint(json.MarshalIndent(list, "", "    "))
+		} else if isCsvOutput() {
+			fmt.Printf("%s\n", strings.Join([]string{"NAME", "ID", "ALIASES"}, ","))
```

Comment:
Not sure if we need something like NewTabWriter after looking at it more closely since we don't need to handle spacing for csv like we do when printing in table format.  I can add it back in but I noticed that the Join seemed to be working fine in this case.